db/migrate/20250208024746_modify_column_to_user.rb

```
@@ -0,0 +1,7 @@
+class ModifyColumnToUser < ActiveRecord::Migration[8.0]
+  def change
+    change_column :users, :age, :integer
```

RuboCop (1.69.1)

[convention] You can use `change_table :users, bulk: true` to combine alter queries.

(Rails/BulkChangeTable)